cards_exporter/cards_pdf_exporter.py

```diff
@@ -0,0 +1,32 @@
+import pdfkit
+from jinja2 import Environment, FileSystemLoader
+from typing import List, NoReturn
+from entities.card import Card
+from cards_exporter.icardexporter import ICardExporter
+from Utils.configuration_keys import ConfigurationKeys
+
+
+class CardPdfExporter(ICardExporter):
+
+    def __init__(self, configuration: dict):
```

Comment:
add type of the variables in the key and value, you welcome to ask if you dont know how to :)